### ISSUE TITLE
_bouncecolorscale seems to have no effect

### ISSUE THREAD

**Issue by 4LT** [ericwa/ericw-tools#424]
A light source adjacent to an orange wall seems to have no effect on the tint of a nearby off-white wall.  Not sure if this is because the orange wall is too dark or what, but I can't detect a hint of orange on the white wall.

![lim_4lt_2024-05-06_23-06-36](https://github.com/ericwa/ericw-tools/assets/3385021/9fc0d391-9e6e-49ba-aafb-1104d6195730)
![lim_4lt_2024-05-06_23-06-42](https://github.com/ericwa/ericw-tools/assets/3385021/5aa45197-5bcf-45e1-a081-48b19331f41e)

Settings:
```
_bounce 3
_bouncecolorscale 1
```

**Comment by 4LT** [comment]
I added some debug prints, `light_options.bouncecolorscale.value()` is returning `0` in the function `CacheTextures`

**Comment by ericwa** [comment]
Cool, I think I can repro this if I set `_bouncecolorscale 1` in worldspawn.
Previously I was setting `-bouncecolorscale 1` as a light command-line arg.

Does it fix it if you pass `-bouncecolorscale 1` as a light command line arg?

**Comment by 4LT** [comment]
Yes, it works now!  Although the effect is much more subtle than I had hoped for
